src/sockperf.cpp

```diff
@@ -328,6 +328,9 @@ static const AOPT_DESC client_opt_desc[] = {
       aopt_set_literal(0),
       aopt_set_string("increase_output_precision"),
       "Increase number of digits after decimal point of the throughput output (from 3 to 9). " },
+    { OPT_TCP_NB_CONN_TIMEOUT_MS,                            AOPT_ARG,
+      aopt_set_literal(0),                             aopt_set_string("tcp_nb_connect_timeout_ms"),
+      "Non-blocking connect timeout in milliseconds." },
```

Comment:
I prefer to provide default value in help output

Reply:
Done